### PR TITLE
added languages for API

### PR DESCRIPTION
**Pull request by SurfaceS**
Add all languages for the API.
Then the UMS player have translated strings according to selected language.

### CODE REVIEW

**Comment by SurfaceS** [comment]
Sorry, it is based on the #3648 so it is hard to see the changes.

**Comment by SurfaceS** [comment]
@SubJunk  : To see the real changes, check here :
https://github.com/SurfaceS/UniversalMediaServer/pull/122

**Comment by SurfaceS** [comment]
merged main and added some quick code for the api side to get ride of the local info.movito.themoviedbapi use.

**Comment by SurfaceS** [comment]
modified according to https://github.com/UniversalMediaServer/api/pull/1145

added tmdbId on tables
added reference more readable on tables
refactored some table columns

**Comment by SurfaceS** [comment]
Tested after the api merge.

Works great.
Go on a movie (or series, episode) on player, change language, enjoy.

**Comment by SurfaceS** [comment]
added modified column to tables (future use, to fix [#831](https://github.com/UniversalMediaServer/api/issues/831)).

On react interface :
update mantine -> 5.9.5.
added tmdb attribution on react about page (as asked from rules).
fix react player media page title.
added tmdb link to react player media page.
added tagline to react player media page.

FYI : 
Most movies/tv episodes stored in the **api database** does not have the tmdb id or tmdb tv id stored.
The api store it only from the last PR.
So even now, UMS requests will not have it if movie/tv episode was found before the [PR](https://github.com/UniversalMediaServer/api/pull/1145).

Most tv shows stored in the **api database** have tmdb id stored.
But UMS database do not store it until this PR, so all current stored tv shows do not have the tmdb id.

Basically, that mean that tmdb link to react player media page will be visible on few media for now.

**Comment by SurfaceS** [comment]
should be done.

File: src/main/java/net/pms/database/MediaTableTVSeries.java [hunk 1]
```diff
@@ -737,6 +782,18 @@ public static JsonObject getTvSeriesMetadataAsJsonObject(final Connection connec
 						result.addProperty("rating", MediaTableVideoMetadataIMDbRating.getValueForTvSerie(connection, id));
 						result.add("ratings", MediaTableVideoMetadataRatings.getJsonArrayForTvSerie(connection, id));
 						result.addProperty("released", MediaTableVideoMetadataReleased.getValueForTvSerie(connection, id));
+						if (lang != null && !"en-us".equalsIgnoreCase(lang)) {
+							VideoMetadataLocalized loc = MediaTableVideoMetadataLocalized.getVideoMetadataLocalized(connection, id, true, lang, imdbID, "tv", tmdbId, null, null);
+							if (loc != null) {
+								loc.localizeJsonObject(result);
+								//temp fix to store tmdbID if it was not before
```
Comment (SubJunk) [inline src/main/java/net/pms/database/MediaTableTVSeries.java:789]: I'm hesitant about temporary fixes since they often stick around as we forget to remove them, and it can be hard to know when it's ok to remove them. If this is really the best strategy, can we leave a comment in the code that details exactly what the plan is, including when this can be removed?

Comment (SurfaceS) [reply src/main/java/net/pms/database/MediaTableTVSeries.java]: In fact, it is not a temp fix as the situation can be found even with the current code.

It test the presence of the tmdb ID, and store it if it was not present.
So it will happen only once.

**Comment by SubJunk** [comment]
I'm having a hard time testing it because when I change the language on the web interface, it gets stuck with a loading overlay that doesn't end

**Comment by SurfaceS** [comment]
> I'm having a hard time testing it because when I change the language on the web interface, it gets stuck with a loading overlay that doesn't end.

It is a bug with rtl not related to this PR.
I never found why it do that.

**Comment by SurfaceS** [comment]
> the language is changed, but I don't see any different metadata.

sorry... bug fixed

**Comment by SurfaceS** [comment]
> I'm having a hard time testing it because when I change the language on the web interface, it gets stuck with a loading overlay that doesn't end.

@SubJunk Found it : the lazy load break the theme things.

Fixed.

**Comment by SubJunk** [comment]
@SurfaceS nice work, I'll test more today

**Comment by SubJunk** [comment]
@SurfaceS this works great now, it's amazing. It's so smooth. I think there will be many happy users with this

**Review by SubJunk — APPROVED** [review]
